soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3888,6 +3899,34 @@ void DrawRandoEditor(bool& open) {
                 ImGui::BeginChild("ChildMiscWorldSettings", ImVec2(0,-8));
                 ImGui::PushItemWidth(-FLT_MIN);
 
+                //Starting Age
+                //Disabled when Forest is set to Closed
+                bool disableRandoStartingAge = !CVar_GetS32("gRandomizeForest", 0);
+                const char* disableRandoStartingAgeText = "This option is disabled because \"Forest\" is set to \"Closed\".";
+                ImGui::Text(Settings::StartingAge.GetName().c_str());
+                UIWidgets::InsertHelpHoverText(
+                    "Choose which age Link will start as.\n\n"
+                    "Starting as adult means you start with the Master Sword in your inventory.\n"
+                    "Only the child option is compatible with Closed Forest."    
+                );
+               if (disableRandoStartingAge) {
+                    ImGui::PushItemFlag(ImGuiItemFlags_Disabled, true);
+                    ImGui::PushStyleVar(ImGuiStyleVar_Alpha, ImGui::GetStyle().Alpha * 0.5f);
+                }    
+                UIWidgets::EnhancementCombobox("gRandomizeStartingAge", randoStartingAge, 3, 0);
+                //    ImGui::PushItemFlag(ImGuiItemFlags_Disabled, true);
+                //    ImGui::PushStyleVar(ImGuiStyleVar_Alpha, ImGui::GetStyle().Alpha * 0.5f);
```

Comment:
What are these commented out for? We should remove them or leave a comment on why they were left